### PR TITLE
SUS-3268 | sunset PowerUser feature

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/SUS-3268

The maintenance script that looks for new power users (i.e. users that have made more than 140 edits in the last 60 days) has been disabled (temporary  ) back in January 2016. The code is enabled site-wide, works, but uses quite outdated information regarding how's a power user.

Database cleanup will follow.

### CODE REVIEW

File: extensions/wikia/AnalyticsEngine/js/universal_analytics.js [hunk 1]
```diff
@@ -422,8 +420,6 @@
 	window.ga('ads.set', 'dimension19', window.wgArticleType);                           // ArticleType
 	window.ga('ads.set', 'dimension20', 'not set');                                      // Performance A/B testing (not used any more)
 	window.ga('ads.set', 'dimension21', String(window.wgArticleId));                     // ArticleId
-	window.ga('ads.set', 'dimension23', window.wikiaIsPowerUserFrequent ? 'Yes' : 'No'); // IsPowerUser: Frequent
-	window.ga('ads.set', 'dimension24', window.wikiaIsPowerUserLifetime ? 'Yes' : 'No'); // IsPowerUser: Lifetime
```
Comment (macbre) [inline extensions/wikia/AnalyticsEngine/js/universal_analytics.js:426]: @nandy-andy - please code review this from AdEngineering / events tracking perspective

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 2]
```diff
@@ -253,8 +251,6 @@ public function getTrackingDimensions() {
 		$dimensions[15] = WikiaPageType::isCorporatePage() ? 'yes' : 'no';
 		$dimensions[17] = WikiFactoryHub::getInstance()->getWikiVertical( $wgCityId )['short'];
 		$dimensions[18] = $wikiCategoryNames;
-		$dimensions[23] = in_array( 'poweruser_lifetime', $powerUserTypes ) ? 'yes' : 'no';
-		$dimensions[24] = in_array( 'poweruser_frequent', $powerUserTypes ) ? 'yes' : 'no';
```
Comment (macbre) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php:257]: @hakubo - please code review this part from Mercury perspective

**Review by mszabo-wikia — APPROVED** [review]

File: extensions/wikia/AnalyticsEngine/js/universal_analytics.js (hunk 1 above)
Comment (nandy-andy) [reply extensions/wikia/AnalyticsEngine/js/universal_analytics.js:426]: Looks fine - I have never used this dimensions and removing them seems OK :)

Is there a way in GA to add a point in the timeline and inform us in future that until the day this pull request was merged dimensions 23 and 24 were connected to power users? ;)

File: extensions/wikia/MercuryApi/MercuryApiController.class.php (hunk 2 above)
Comment (hakubo) [reply extensions/wikia/MercuryApi/MercuryApiController.class.php:257]: lgtm
should we also remove this?
https://github.com/Wikia/mobile-wiki/blob/cb593de357863cf38722e09297da97e4e10579b5/app/models/user.js#L191

Comment (macbre) [reply extensions/wikia/MercuryApi/MercuryApiController.class.php:257]: Thanks for the link - https://github.com/Wikia/mobile-wiki/pull/413

**Pull request merged**